openapi.yaml

```diff
@@ -8,6 +8,79 @@ info:
     email: mail@kopfsachen.org
     url: 'https://www.kopfsachen.org'
 paths:
+  '/self-service/registration/api':
+    get:
+      summary: Initate registration flow
+      tags:
+        - authentication
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/registration'
+        '500':
+          description: Internal Server Error
+  '/self-service/registration':
+    post:
+      parameters:
+        - $ref: '#/components/parameters/flow'
+      summary: Receive accountKey during registration
+      tags:
+        - authentication
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/registration-with-flow-id-response'
+        '500':
+          description: Internal Service Error
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/registration-with-flow-id-request'
+  '/self-service/login/api':
+    get:
+      tags:
+        - authentication
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/registration'
+        '500':
+          description: Internal Service Error
+  '/self-service/login/flow':
```

Comment:
According to the Kratos API docs, this path should just be `/self-service/login`: https://www.ory.sh/docs/kratos/reference/api#operation/submitSelfServiceLoginFlow